### PR TITLE
EVEREST-1160 | Improve API documentation

### PR DESCRIPTION
**Pull request by mayankshah1607**
- Update API tags
- Set tags on untagged APIs
- Update summary
- Update descriptions

### CODE REVIEW

File: docs/spec/openapi.yml [hunk 1]
```diff
@@ -126,9 +147,10 @@ paths:
   '/cluster-info':
     get:
       tags:
-        - k8s
-      summary: Get the cluster type and storage classes of a kubernetes cluster
-      description: Get the cluster type and storage classes of a kubernetes cluster
+        - Kubernetes
+      summary: Cluster info
+      description: |
+        This API gets the type of the cluster type and the storage class used by the cluster.
```
Comment (recharte) [inline docs/spec/openapi.yml:153]:
```suggestion
        This API gets the type of the cluster and the storage classes available in the cluster.
```
or 
```suggestion
        This API gets the cluster type and the storage classes available in the cluster.
```

File: docs/spec/openapi.yml [hunk 2]
```diff
@@ -236,9 +260,10 @@ paths:
   '/namespaces/{namespace}/database-clusters/{name}':
     get:
       tags:
-        - databaseCluster
-      summary: Get the specified database cluster
-      description: Get the specified database cluster
+        - Database Cluster
+      summary: Get database cluster
+      description: |
+        This API gets the specified database cluster specified by the `name` and `namespace`.
```
Comment (recharte) [inline docs/spec/openapi.yml:266]:
```suggestion
        This API gets the database cluster specified by the `name` and `namespace`.
```

File: docs/spec/openapi.yml [hunk 3]
```diff
@@ -319,9 +345,10 @@ paths:
               $ref: '#/components/schemas/DatabaseCluster'
     delete:
       tags:
-        - databaseCluster
-      summary: Delete the specified database cluster
-      description: Delete the specified database cluster
+        - Database Cluster
+      summary: Delete database cluster
+      description: |
+        This API deletes the specified database cluster specified by the `name` and `namespace`.
```
Comment (recharte) [inline docs/spec/openapi.yml:351]:
```suggestion
        This API deletes the database cluster specified by the `name` and `namespace`.
```

File: docs/spec/openapi.yml [hunk 4]
```diff
@@ -480,9 +510,10 @@ paths:
   '/namespaces/{namespace}/database-clusters/{name}/pitr':
     get:
       tags:
-        - databaseCluster
-      summary: Get the Point-in-Time related data for the specified database cluster
-      description: Get the Point-in-Time related data for the specified database cluster
+        - Database Cluster
+      summary: Get the Point-in-Time info
+      description: |
+        This API gets the Point-in-Time info for the database cluster specified by the `name` and `namespace`.
```
Comment (recharte) [inline docs/spec/openapi.yml:516]:
```suggestion
      summary: Get the Point-in-Time recovery info
      description: |
        This API gets the Point-in-Time recovery info for the database cluster specified by the `name` and `namespace`.
```

File: docs/spec/openapi.yml [hunk 5]
```diff
@@ -720,9 +756,10 @@ paths:
   '/namespaces/{namespace}/database-engines/{name}/operator-version':
     get:
       tags:
-        - getOperatorVersion
-      summary: Get the operator version and check results for the specified database engine
-      description: Get the operator version and check results for the specified database engine
+        - Operators
+      summary: Version info
+      description: |
+        This API gets the operator version and check results for the specified database engine.
```
[the remark this answers is not in the document]
Comment (mayankshah1607) [reply docs/spec/openapi.yml]: Updated

**Review by recharte — APPROVED** [review]
:star2:

**Pull request merged**